routes/routes.js

```diff
@@ -0,0 +1,12 @@
+const router = require('express').Router();
```

Comment:
Можно лучше: точку входа в группу модулей именовать `index.js`.
`index.js` является базовым файлом, который будет подключен по-умолчанию, если будет произведено подключение модуля. Например, если переименовать, то можно подключить роуты вот так:
```suggestion
const routes = require('./routes');
```

Как минимум стало короче.